README.md

```diff
@@ -1,30 +1,100 @@
-# PegNet
-This is now the main repository for mining for the PegNet
+<p align="center">
+  <img src="https://pegnet.org/assets/img/logo.png"/>
+</p>
 
-The TestNet has started writing records in mid June.
+----
 
-This is a work in progress.  The TestNet is certain to reset the actual rewards a few times as we develop more code and adjust the approach.  None the less, it is a great help to have multiple people do mining.  If you would like to help, you can run a miner.  To do so, do these 10 things:
+[![Build Status](https://travis-ci.com/pegnet/pegnet.svg?branch=develop)](https://travis-ci.com/pegnet/pegnet)
+[![Discord](https://img.shields.io/discord/550312670528798755.svg?label=&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2)](https://discord.gg/V6T7mCW)
+[![License](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/pegnet/pegnet/blob/master/LICENSE)
 
-*Assumes Linux or use of a virtual machine running Linux*
 
-1. Clone the OracleRecord repository https://github.com/pegnet/pegnet
 
-2. Copy the defaultconfig.ini file from the OracleRecord repository to ~/.pegnet directory
+## A Network of Pegged Tokens
 
-3. Sign up for an API Key from https://currencylayer.com.
+This is the main repository for the PegNet application.
 
-4. Replace the Oracle.APILayerKey key with your API Key
+Pegged tokens reflect real market assets such as currencies, precious metals, cryptocurrencies, commodities, etc. The conversion rates on PegNet are determined by a decentralized set of miners who submit values based on current market data. These values are then graded and recorded in the Factom blockchain. 
 
-5. Edit the Miner.IdentityChain to add a unique name like:
+The draft proposal paper is available [here](https://docs.google.com/document/d/1yv1UaOXjJLEYOvPUT_a8RowRqPX_ofBTJuPHmq6mQGQ).
 
-6. IdentityChain=prototype,BigMiner
+For any questions, troubleshooting or further information head to [discord](https://discord.gg/V6T7mCW).
 
-7. Right now you can donate to the cause, and leave the pPNT address alone. We will had a utility to generate your own PNT address from one of your FCT addresses in your wallet
+All mining is currently done on Testnet.
```

Comment:
more clarification? it runs on the *PegNet TestNet* which runs on the Factom MainNet

Reply:
By the time this leaves dev branch the statement is probably redundant. A half hearted disclaimer if you will?